api/envoy/config/bootstrap/v3/bootstrap.proto

```diff
@@ -60,6 +61,9 @@ message Bootstrap {
     // These static secrets can be used by :ref:`SdsSecretConfig
     // <envoy_api_msg_extensions.transport_sockets.tls.v3.SdsSecretConfig>`
     repeated envoy.extensions.transport_sockets.tls.v3.Secret secrets = 3;
+
+    // Allows loading of multiple socket interfaces and configuration of the default one
+    core.v3.SocketInterfacesConfig socket_interfaces = 4;
```

Comment:
Is the thinking that these will one day be dynamically discovered by an xDS? If not, you don't need to put them in `static_resources`, they can just live top-level.

Reply:
Done! Actually this was just me not knowing what I was doing, so thanks for the comment!

Another thing that's confusing is the ci/circleci:docs error. Should I add the new *.proto files to some list or is there something wrong with them?